hack/e2e.sh

```diff
@@ -29,6 +29,23 @@ set -o pipefail
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 cd "${REPO_ROOT}" || exit 1
 
+# If kind is not available on the path, get it
+if ! [ -x "$(command -v kind)" ]; then
+  echo 'kind not found, installing'
+  pushd ~
+  GOFLAGS="" GO111MODULE="on" go get sigs.k8s.io/kind@v0.4.0GOFLAGS="" GO111MODULE="on" go get sigs.k8s.io/kind@v0.4.0
```

Comment:
This line looks like a copy/paste error (2x)?

Reply:
Indeed